Source/sound.h

```diff
@@ -30,7 +38,7 @@ void sound_file_cleanup(TSnd *sound_file);
 void snd_init();
 void sound_cleanup();
 void music_stop();
-void music_start(int nTrack);
+void music_start(uint8_t nTrack);

```

Comment:
Can we do this?
```suggestion
void music_start(_music_id nTrack);
```

Reply:
I initially did but it gets fed `leveltype` in some cases and tracks are switched using ++ so it ended up being an unholy casting mess, probably some additional refactoring would be good here.